dove/src/cli.rs

```diff
@@ -162,6 +174,7 @@ pub fn execute(args: Vec<String>, cwd: PathBuf) -> Result<()> {
         DoveCommands::Run { mut cmd } => cmd.apply(&mut ctx),
         DoveCommands::Call { mut cmd } => cmd.apply(&mut ctx),
         DoveCommands::Deploy { mut cmd } => cmd.apply(&mut ctx),
+        DoveCommands::Key { mut cmd } => cmd.apply(&mut ctx),
```

Comment:
`DoveCommands::Key` is already handled at this point, see how `DoveCommands::Clean` is treated in this match.